graphite_api/app.py

```diff
@@ -460,10 +460,9 @@ def render():
             series_data = {}
             labels = ['Time']
             if any(context['data']):
-                datapoints = [[ts * 1000]
-                              for ts in range(context['data'][0].start,
-                                              context['data'][0].end,
-                                              context['data'][0].step)]
+                datapoints = [[t * 1000] for t in range(context['data']
+                              [0].start, context['data'][0].end,
+                              context['data'][0].step)]
```

Comment:
linting was failed in my case because of this, I will return it back original version.